pystiche_papers/sanakoyeu_et_al_2018/_modules.py

```diff
@@ -48,12 +37,22 @@ def conv(
     in_channels: int,
     out_channels: int,
     kernel_size: Union[Tuple[int, int], int],
-    stride: Union[Tuple[int, int], int] = 2,
```

Comment:
@jbueltemeier I don't know why this was `2` by default? Was there a reason for this? Tests are passing so it seems this is irrelevant for all other internal functions.

Reply:
No, there was no particular reason.